src/component/bookList/BookList.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react'
+
+import { IBook } from '../../interface/IBook'
+import { BookItem } from './BookItem'
+
+import './style.css'
+
+type Props = {
+    books: IBook[]
+}
+
+export const BookList: FC<Props> = ({books}) => {
+
+    return(
+        <div className="wrapperList">
```

Comment:
wrapperBookList - entity must be identified

Reply:
fixed